tests/runner/test_runner.py

```diff
@@ -48,6 +48,13 @@
 from prefect.utilities.filesystem import tmpchdir
 
 
+def make_minimal_deployment_schedule(**kwargs) -> MinimalDeploymentSchedule:
+    """dont raise on `PrefectDeprecationWarning` about using `DeploymentScheduleCreate`"""
```

Comment:
ah so if this is about avoiding the deprecation warnings in tests, you can do this through config instead of code here: https://github.com/PrefectHQ/prefect/blob/main/setup.cfg#L50 (same file for 2.x branch)